addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -18,6 +18,7 @@
 from odoo.tests.common import users
 from odoo.tools import email_split_and_format, formataddr, mute_logger
 
+from email.header import Header
```

Comment:
I'm sorry to be this guy but can you move this import next to the `unittest` ones?

```py
...
import socket

from email.header import Header
from unittest.mock import DEFAULT
from unittest.mock import patch
...

Reply:
and since you're at it, you can change the commit title to `[FIX] core: add support for windows-874 encoding`

Reply:
WDM if it's better/cleaner it should be moved. Thanks!